helm/choreo-connect/templates/gateway-runtime-deployment.yaml

```diff
@@ -113,6 +113,18 @@ spec:
             - mountPath: /home/wso2/check_health.sh
               name: grpc-probe-health-check
               subPath: "check_health.sh"
+            {{- range .Values.wso2.deployment.volumeConfigs }}
+            {{- if eq .mount.container "choreo-connect-enforcer" }}
+            - name: {{ .name }}
+              mountPath: {{ .mount.path }}
```

Comment:
As a practice, its better to include the file also in the mount path to avoid issues when mounting multiple files to the same location.

```
- name: enforcer-volume
  mountPath: /home/wso2/conf/sample-config-map/test.yaml
  subPath: test.yaml
```

Reply:
Fix [ab215e0](https://github.com/wso2/kubernetes-microgateway/pull/115/commits/ab215e047091cabec9b2e1aef18f1c58bfe16d77). Updated the doc in README to communicate this to customers when mounting their files